website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another WSL distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, Podman Desktop manages Podman in a dedicated Windows Subsystem for Linux (WSL) distribution: the Podman machine.
+
+For your usual WSL workloads, you might want to continue working with your usual WSL distribution, and access from there to your Podman Desktop containers.
+
+This tutorial focuses on Ubuntu, the default WSL distribution, to walk you through the steps to configure your WSL distribution:
+
+1. Install the Podman client.
+2. Give access to the Podman machine.
+3. Assign the remote destination.
+4. Test the connection.
+
+## Installing the Podman client
+
+On your WSL distribution, install the Podman client to send requests to the Podman machine.
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. Install the Podman client:
+
+   ```shell-session
+   $ sudo apt install podman
+   ```
+
+   The package might set up local Podman resources, but you are not actively using them.
+
+## Giving access to the Podman machine
+
+On your WSL distribution, give access to the remote Podman machine to your user, by assigning group membership.
```

Comment:
Note that this step would not be necessary if the socket was assigned to the group `1000`.

Reply:
```suggestion
On your WSL distribution, give access to the remote Podman machine to your user, by assigning group membership.

It will allow your user to access a special file (a socket), that is the communication channel to the Podman Machine.
```